src/Model/Consignment/DHLForYouConsignment.php

```diff
@@ -38,6 +38,7 @@ public function getAllowedPackageTypes(): array
         return [
             self::PACKAGE_TYPE_PACKAGE_NAME,
             self::PACKAGE_TYPE_MAILBOX_NAME,
+            self::PACKAGE_TYPE_PACKAGE_SMALL_NAME
```

Comment:
Wij eindigen elke regel graag met een comma sinds dat mag in een array in php, dat is makkelijker copy-pasten.